open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/util/impl/DataTypeActionUtils.java

```diff
@@ -41,16 +41,17 @@ public static void gotoDataType(DataTypeInfo dataType, DynamicViewer viewer)
     public static void gotoLocations(Collection<? extends LatLonAlt> locations, DynamicViewer viewer)
     {
         GeographicBoundingBox bbox = GeographicBoundingBox.getMinimumBoundingBoxLLA(locations);
-        gotoBoundingBox(bbox, viewer);
+        gotoBoundingBox(bbox, viewer, true);
     }
 
     /**
      * Goes to the bounding box.
      *
      * @param bbox the bounding box
      * @param viewer the viewer
+     * @param flyTo the flyTo animation
```

Comment:
Let's explain what this is better, like true to fly to, and false to snap to, or something like that.